Simple Incremental/Assets/Scripts/Movement.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class Movement : MonoBehaviour
+{
+
+    public event MovementFinishedHandler OnMovementFinished;

```

Comment:
Where is this going to be consumed?

Reply:
Maybe some enemies will start attacking only when they are in melee range. Or thye might have some ability that triggers once they are done moving.

Reply:
This makes sense.